src/ufoLib2/objects/font.py

```diff
@@ -1,20 +1,24 @@
 import os
 import shutil
-from typing import Any, Dict, List, Mapping, Optional, Tuple, Union
+from typing import Any, Dict, Iterator, KeysView, List, Mapping, Optional, Tuple, Union
 
 import attr
+import fs
 import fs.tempfs
 from fontTools.ufoLib import UFOFileStructure, UFOReader, UFOWriter
 
 from ufoLib2.constants import DEFAULT_LAYER_NAME
+from ufoLib2.errors import Error
 from ufoLib2.objects.dataSet import DataSet
 from ufoLib2.objects.features import Features
+from ufoLib2.objects.glyph import Glyph
 from ufoLib2.objects.guideline import Guideline
 from ufoLib2.objects.imageSet import ImageSet
 from ufoLib2.objects.info import Info
+from ufoLib2.objects.layer import Layer
 from ufoLib2.objects.layerSet import LayerSet
 from ufoLib2.objects.misc import _deepcopy_unlazify_attrs
-from ufoLib2.typing import Number
+from ufoLib2.typing import Number, PathLike, T
```

Comment:
What does it mean to import this `T` from another file? Should you do that? (sorry I don't really get how the type variables are supposed to be used in Python)

Reply:
I just put it there because I'm using `TypeVar("T")` in a bunch of places that have a `get` method🤷‍♀️ It works, so, uh...

Reply:
Ok I guess :) These type variables floating around as real objects look strange to me compared to other languages with generics

Reply:
Yeah. Such is life when a type system is tacked on I guess.